.coveragerc

```diff
@@ -0,0 +1,4 @@
+[run]
+  source = serenata_toolbox
+  omit =
```

Comment:
Sure! I can do this right away! I would just like to quickly remind that the `*/virtualenv/*` rule was used because of the libraries included by Travis during the build process, which I believe to be the source of the problem:

<img width="738" alt="screen shot 2017-05-24 at 2 48 59 pm" src="https://cloud.githubusercontent.com/assets/4705127/26418010/1cf2e000-4091-11e7-82a3-aae4e48e9256.png">

This is slightly irrelevant for this issue's solution since the `source` section of the config file seems to be all we need to fix the problem (I'll update this PR accordingly as soon as all the tests are finished).